<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnswerRouteSearch</class>
 <widget class="QWidget" name="AnswerRouteSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>144</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(39, 100, 54);
font: 60pt &quot;Oswald&quot;;
border-bottom: 1px solid #000000;</string>
   </property>
   <property name="text">
    <string>Recommended Route</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>816</y>
     <width>1280</width>
     <height>144</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 30pt &quot;Oswald&quot;;
	border-bottom-left-radius:5px;
	border-bottom-right-radius:5px;
	border-top: 1px solid #000000;
}

QPushButton::Hover{
	border-color: rgb(0, 0, 0);
	border: 5px solid #000000;
}

QPushButton::Pressed{
	border-color: rgb(0, 0, 0);
	border: 5px solid;
	background-color: rgb(90,152,101);
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="Text1">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>204</y>
     <width>816</width>
     <height>96</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 30pt &quot;Oswald&quot;;
	border-radius: 20px;
	border: 1px solid #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Text2">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>360</y>
     <width>544</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 30pt &quot;Oswald&quot;;
	border-radius: 20px;
	border: 1px solid #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Text3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>360</y>
     <width>544</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 30pt &quot;Oswald&quot;;
	border-radius: 20px;
	border: 1px solid #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="EndRoute">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>660</y>
     <width>544</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 30pt &quot;Oswald&quot;;
	border-radius: 20px;
	border: 1px solid #000000;
}

QPushButton::Hover{
	border-color: rgb(0, 0, 0);
	border: 5px solid #000000;
}

QPushButton::Pressed{
	border-color: rgb(0, 0, 0);
	border: 5px solid;
	background-color: rgb(90,152,101);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="RecRoute">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>660</y>
     <width>816</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 30pt &quot;Oswald&quot;;
	border-radius: 20px;
	border: 1px solid #000000;
}

QPushButton::Hover{
	border-color: rgb(0, 0, 0);
	border: 5px solid #000000;
}

QPushButton::Pressed{
	border-color: rgb(0, 0, 0);
	border: 5px solid;
	background-color: rgb(90,152,101);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="StartRoute">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>660</y>
     <width>544</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 30pt &quot;Oswald&quot;;
	border-radius: 20px;
	border: 1px solid #000000;
}

QPushButton::Hover{
	border-color: rgb(0, 0, 0);
	border: 5px solid #000000;
}

QPushButton::Pressed{
	border-color: rgb(0, 0, 0);
	border: 5px solid;
	background-color: rgb(90,152,101);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
